src/OrchardCore.Modules/OrchardCore.ContentFields/Views/TextFieldPredefinedListEditorSettings.Edit.cshtml

```diff
@@ -8,7 +8,7 @@
 <script asp-src="https://cdnjs.cloudflare.com/ajax/libs/Vue.Draggable/2.14.1/vuedraggable.min.js" debug-src="https://cdnjs.cloudflare.com/ajax/libs/Vue.Draggable/2.14.1/vuedraggable.js" asp-name="vuedraggable" depends-on="vuejs, sortable" at="Foot"></script>
 
 <script at="Foot">
-    initializeOptionsEditor(document.getElementById('@Html.IdFor(m => m)'), @Html.Raw(Model.Options), document.getElementById('@Html.IdFor(m => m.Options)'));
+    initializeOptionsEditor(document.getElementById('@Html.IdFor(m => m)'), @Html.Raw(Model.Options), '@Model.DefaultValue');
```

Comment:
You can't inject a value inside javascript strings, the value could have javascript injection. You need to inject it in a hidden div's attribute (like we do for javascript translations) and use a jquery selector to get the value here.

Reply:
I'm trying different things to understand how Vue.js works with data values. I think the issue is that I'm specifying the input type="radio" name="@Something" and using also v-model="defaultValue" which makes the checked item to not persist. In fact it's working fine when I load the page and that I've not saved any defaultValue. The only issue remaining is happening when you load the page after saving a defaultValue and that you select a different default value than the one selected and that afterward you reorder the items. It will persist to the default value that was loaded initially.

Reply:
Pushed my latest code. What I'm saying in the previous comment is not working either. I'm going to look at it later today.